utils/unittest/fixtures.go

```diff
@@ -375,14 +375,15 @@ func ExecutionReceiptFixture() *flow.ExecutionReceipt {
 }
 
 func ExecutionResultFixture() *flow.ExecutionResult {
+	blockID := IdentifierFixture()
 	return &flow.ExecutionResult{
 		ExecutionResultBody: flow.ExecutionResultBody{
 			PreviousResultID: IdentifierFixture(),
-			BlockID:          IdentifierFixture(),
+			BlockID:          blockID,
 			FinalStateCommit: StateCommitmentFixture(),
 			Chunks: flow.ChunkList{
-				ChunkFixture(),
-				ChunkFixture(),
+				ChunkFixture(blockID),
+				ChunkFixture(blockID),
```

Comment:
It's a bit redundant to include blockID to every single chunk, given that all chunks come along with the receipt, and they should all have the same block ID, which is specified by the block ID included in the result of the receipt. (I'm commenting here, because this test case makes it obvious)

In other word, I think there are alternatives that we could compute the chunk ID without attaching block ID to each chunk. 

I agree including the blockID here is a quick and easy fix, but it increases the message size and also introduce the need to validate them - as all chunks should have the same block ID as the receipt includes.

Did we evaluate other alternatives?

Reply:
@zhangchiqing including the block ID into the chunks was decided in [this Technical Product Meeting](https://www.notion.so/dapperlabs/Technical-Product-Meeting-with-Dete-054824f0be334392806e1e39641afe48#03a5189cb5fa42cebeeb7357cafc0382). As far as I remember from the meeting, its purpose is more than the chunk ID itself. As discussed there, Including the block ID into the chunk, body makes the chunks self identified as well using `(chunk index, block id)` pairs.